package.json

```diff
@@ -29,10 +29,16 @@
   "homepage": "https://github.com/marlon360/nifty-uploader#readme",
   "devDependencies": {
     "@types/jest": "^23.3.10",
+    "cz-conventional-changelog": "^2.1.0",
```

Comment:
I think this is not relevant for this PR.
And I think we do not need it.

Reply:
This is a tool for me to format git commit. I will delete it.